rustql-extractor/Cargo.toml

```diff
@@ -8,13 +8,14 @@ license = "MIT/Apache-2.0"
 edition = "2018"
 
 [[bin]]
-name = "rustc"
+name = "rustql-extractor"
```

Comment:
Have you tried running with [sccache](https://github.com/mozilla/sccache)? At least `sccache` use to assume that the binary is called `rustc`.

Reply:
I hadn't tried that. I tried and is still needed so I changed it.